README.md

```diff
@@ -1,97 +1,49 @@
-
-#### Build
-
-The task `build` compiles and builds the whole project. The result will be stored in `dist/`.
-
-#### Testing
-
-Via the task `test` code can be tested with [Karma](https://karma-runner.github.io).
-
-#### End-to-End Testing
-
-End-to-end tests can be made with `e2e` and [Protractor](http://www.protractortest.org/).
-
-### Debugging
-
-VSCode comes with a build-in [debugger](https://code.visualstudio.com/Docs/editor/debugging). It's already configured. The only thing you have to do is running the taks `run`, switching to the debug section and starting `Run`.
-
-**!** For the debugger the extension _Debugger for Chrome_ must be installed.
-
-## The Angular CLI
-
-The Angualar CLI helps you managing Angular projects.
-
-### Generating Components
-
-The command `ng generate component component-name` (short: `ng g c component-name`) creates a new Component. Details can be found in the Angular documentation.
-
-### Build
-
-`ng build` compiles and builds the whole project. The result will be stored in `dist/`. For production mode use the `--prod` argument.
-
-### Testing
-
-Via `ng test` code can be tested with [Karma](https://karma-runner.github.io).
-
-### End-to-End Testing
-
-End-to-end tests can be made with `ng e2e` and [Protractor](http://www.protractortest.org/).
+VSCode allows to create tasks, which can be very helpful to manage big projects without a _CLI_. Tasks can be executed via the command palette (open with _Ctrl+Shift+P / CMD+Shift+P_). Learn more in this [task tutorial](https://code.visualstudio.com/Docs/editor/tasks).
 
-## Appendix
+Tasks:  
 
-### Advice for browser compatibility
+- `build`: compiles and builds the whole project. The result will be stored in `dist/`.
+- `test`: test your code with [Karma](https://karma-runner.github.io).
 
-Every contributor should test his code minimum with one WebKit browser, such as Chrome, Chromium, Safari and Opera, and a non-WebKit browser, such as Firefox, Internet Explorer or Edge.
+## Build & Test
 
-Before you use very new or old things in your code it's recommendet to use "[Can I use](https://caniuse.com/)". With this tool you can test the compatibility of your code.
+- `ng build`: compiles and builds the whole project. The result will be stored in `dist/`. For production mode use the `--prod` argument.
+- `ng test`: tests your code with [Karma](https://karma-runner.github.io).
+- `ng e2e`: runs end-to-end tests with [Protractor](http://www.protractortest.org/).
 
-[CSS tricks](https://css-tricks.com) delivers tipps for CSS/SCSS (e.g. centering, flex-box, grid etc.).
+## Check browser compatibility
 
-### Further help
+To check browser-compatibility, use [Can I use](https://caniuse.com/). You should test your code at minimum with one WebKit browser (Chrome, Chromium, Safari, or Opera) and one non-WebKit browser (Firefox, Internet Explorer, or Edge). [CSS tricks](https://css-tricks.com) delivers tips for CSS/SCSS (e.g. centering, flex-box, grid etc.).
```

Comment:
Angular dropped support for IE & move Edge to WebKit

Reply:
Put the css help in a seperate line. (`<br>` - break)

Reply:
```suggestion
To check browser-compatibility, use [Can I use](https://caniuse.com/). You should test your code at minimum with one WebKit browser (Chrome, Chromium, Safari, or Opera) and one non-WebKit browser (Firefox or Microsoft Edge). 

Note: [CSS tricks](https://css-tricks.com) delivers tips for CSS/SCSS (e.g. centering, flex-box, grid etc.).
```